tests/dialects/test_redshift.py

```diff
@@ -101,7 +101,22 @@ def test_redshift(self):
         self.validate_all(
             "SELECT DISTINCT ON (a) a, b FROM x ORDER BY c DESC",
```

Comment:
Note: this is parsed with Redshift as the input dialect, which displays the `NULL`s first with the `DESC` order.

<img width="763" alt="Screenshot 2023-05-03 at 4 19 03 PM" src="https://user-images.githubusercontent.com/46752250/235927645-462a653b-c12a-4dd9-882b-45f8e941ef0a.png">